lib/elixir/test/elixir/kernel/cli_test.exs

```diff
@@ -62,86 +68,105 @@ defmodule Kernel.CLITest do
 end
 
 defmodule Kernel.CLI.ExecutableTest do
-  use ExUnit.Case, async: true
+  use ExUnit.Case,
+    async: true,
+    parameterize:
+      if(PathHelpers.windows?(),
+        do: [%{cli_extension: ".bat"}, %{cli_extension: ".ps1"}],
+        else:
+          [%{cli_extension: ""}] ++
+            if(System.find_executable("pwsh"), do: [%{cli_extension: ".ps1"}], else: [])
+      )
```

Comment:
You don't need to duplicate `parameterize`. Instead, you can define a variable outside of the `defmodule`:

```
test_parameters =       if(PathHelpers.windows?(),
        do: [%{cli_extension: ".bat"}, %{cli_extension: ".ps1"}],
        else:
          [%{cli_extension: ""}] ++
            if(System.find_executable("pwsh"), do: [%{cli_extension: ".ps1"}], else: [])
      )
```

And then do `parameterize: test_parameters` in all of the relevant ones.